src/ComparisonDetails/ComparisonDetails.tsx

```diff
@@ -5,13 +5,18 @@ import { Comparison } from '../comparisonData'
 
 
 const ComparisonDetails = (prop: { selection?: Comparison }) => {
-  return(
-      <section>
-        <p>This is ComparisonDetails Component</p>
-        <p>{prop.selection?.data.title}</p>
-        <p>{prop.selection?.data.deaths}</p>
-      </section>
-  )
+  const verifyProps = () => {
+    if (prop.selection) {
+      return(
+        <section>
+          <p>This is ComparisonDetails Component</p>
+          <p>{prop.selection?.data.title}</p>
+          <p>{prop.selection?.data.deaths}</p>
+        </section>
```

Comment:
Our next goal should be to build out this section and create some sort of boilerplate message that is created rather than just a couple numbers.